pkg/kubectl/clusterrolebinding_test.go

```diff
@@ -0,0 +1,215 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/apis/rbac"
```

Comment:
Please use "k8s.io/api/rbac/v1beta1". I believe you will need to change clusterrolebinding.go as well.
Switch to use versioned type will be the direction we are heading to to decouple kubectl from main k8s repo.

Reply:
Done. PTAL.

Reply:
Now rbac has been upgraded to v1, should we use the newest version?
@mengqiy

Reply:
Good to know. Thanks for pointing it out!
I was checking https://github.com/kubernetes/api/tree/master/rbac which is out of date with https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/api/rbac.
I guess our publish bot is not working well to sync them.